package.json

```diff
@@ -1,14 +1,26 @@
 {
   "name": "covid-tracker-react-capstone",
   "version": "0.1.0",
+  "homepage": "https://benjp009.github.io/covid-tracker-react-capstone/",
   "private": true,
   "dependencies": {
+    "@react-icons/all-files": "^4.1.0",
+    "@reduxjs/toolkit": "^1.7.2",
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.25.0",
+    "prop-types": "^15.8.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-icons": "^4.3.1",
+    "react-redux": "^7.2.6",
+    "react-router": "^6.2.1",
+    "react-router-dom": "^6.2.1",
     "react-scripts": "5.0.0",
+    "redux": "^4.1.2",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.4.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
- [ ] When I run npm start it open the http://localhost:3000/covid-tracker-react-capstone but the page just show a background color to pink. If you remove the 'covid-tracker-react-capstone' from the URL it works 😉

![Screen Shot 2022-02-04 at 10 28 40](https://user-images.githubusercontent.com/67211919/152565899-010ef692-a45f-4f86-84eb-b712562e4596.png)